### PR TITLE
TEL-23 refactored xpath method

### PR DESCRIPTION
**Pull request by anurgali**

### CODE REVIEW

File: src/test/java/HomePageTest.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ public void setUpInternal(){
     public void signInCheck(){
         sleep(1000);
         try {
-            WebElement logOut = driver.findElement(By.xpath("//span[@xpath='1']"));
+            WebElement logOut = findByXPath("//span[@xpath='1']");
```
Comment (anurgali) [inline src/test/java/HomePageTest.java:21]: I think we need to fix this another way.

**Review by kvlanna — APPROVED** [review]

**Review by GoKate — APPROVED** [review]

**Review by AndreiRai — COMMENTED** [review]
# #

- [ ] 

> ###   #@_``_

**Comment by AndreiRai** [comment]
Would it be possible for you to assign me tasks in Dashboard too, even if I haven't been to a lecture?
Thank you.

**Review by AndreiRai — COMMENTED** [review]
Interesting lesson.

**Pull request merged**